docs/reference/Scheduler.md

```diff
@@ -64,7 +64,7 @@ portRange:
   end: 60000
 maxSurge: 30%
 spec:
-  terminationGracePeriod: '100'
```

Comment:
Isn't this a breaking change? What will happen with the current created schedulers?

Reply:
Never mind, this is only the documentation.